documentation/prometheus-mixin/dashboards.libsonnet

```diff
@@ -91,6 +91,63 @@ local g = import 'grafana-builder/grafana.libsonnet';
           { yaxes: g.yaxes('ms') } +
           g.stack,
         )
-      )
-  },
+      ),
+    # Remote write specific dashboard.
+    'prometheus-remote-write.json':
+      g.dashboard('Prometheus Remote Write')
+        .addMultiTemplate('instance', 'prometheus_build_info', 'instance')
+          .addRow(
+            g.row('Timestamps')
+            .addPanel(
+              g.panel('Highest Timestamp In vs. Highest Timestamp Sent') +
+              g.queryPanel('prometheus_remote_storage_highest_timestamp_in_seconds{instance=~"$instance"} - ignoring(queue) group_right(instance) prometheus_remote_storage_queue_highest_sent_timestamp_seconds{instance=~"$instance"}', '{{instance}}-{{queue}}') +
+              { yaxes: g.yaxes('s') }
+            )
+            .addPanel(
+              g.panel('Rate[5m]') +
+              g.queryPanel('rate(prometheus_remote_storage_highest_timestamp_in_seconds{instance=~"$instance"}[5m])  - ignoring (queue) group_right(instance) rate(prometheus_remote_storage_queue_highest_sent_timestamp_seconds{instance=~"$instance"}[5m])', '{{instance}}-{{queue}}')
+            )
+          )
+          .addRow(
+            g.row('Samples')
+            .addPanel(
+              g.panel('Samples In vs. Samples Out') +
+              g.queryPanel('prometheus_remote_storage_samples_in_total{instance=~"$instance"} - ignoring (queue) group_right(instance) prometheus_remote_storage_succeeded_samples_total{instance=~"$instance"}', '{{instance}}-{{queue}}')
```

Comment:
This is still math on a bare counter. I guess we can be pretty sure that those two counters are always reset at the same time, but then we would rely on an implementation detail that is left open by the spec. (Counters may be reset at any time, even without a binary restart.)

The semantics of the panel is also a bit weird as it is implicitly "since binary restarted". Howe about using `rate` or `increase` over a timeframe that makes sense (1h? 1d?)?

Reply:
Note: After some in-person discussion, we decided to not have this particular panel at all.